### ISSUE TITLE
Get a wrong score greater than 1

### ISSUE THREAD

**Issue by Yikun** [ossf/criticality_score#68]
As described in the criticality score introduction:

> It is a number between 0 (least-critical) and 1 (most-critical).

that means the score is [0, 1], but I got score *1.01543*.

```
$ criticality_score  --repo https://github.com/kubernetes/kubernetes
name: kubernetes
url: https://github.com/kubernetes/kubernetes
language: Go
created_since: 80
updated_since: 0
contributor_count: 3708
org_count: 10
commit_frequency: 96.4
recent_releases_count: 79
updated_issues_count: 5065
closed_issues_count: 2835
comment_frequency: 5.6
dependents_count: 424183
criticality_score: 1.01543
```

The reason of score > 0 is that we have the *negative weight* (such as, updated_since weight is -1), 
1. if we just only summarize all weight value, will make the numerator is greater than the denominator.
https://github.com/ossf/criticality_score/blob/2757693995c6c47e0792d524660c6e8b8e905e10/criticality_score/run.py#L426

2. On the other hand, the negative weight also causes to generate the negative score, because the numerator may be less than 0.
https://github.com/ossf/criticality_score/blob/2757693995c6c47e0792d524660c6e8b8e905e10/criticality_score/run.py#L436

**Comment by Yikun** [comment]
Looks like we have two potential solutions:

**Solution 1. make > 1 criticality_score to 1, and make negetive criticality_score to 0** https://github.com/ossf/criticality_score/pull/69

**Solution 2. only summarize all positive weight in denominator, make negetive criticality_score to 0**https://github.com/ossf/criticality_score/pull/70

I prefer solution 1.

**Comment by inferno-chromium** [comment]
```
$ python3 -u -m criticality_score.run --repo github.com/kubernetes/kubernetes
name: kubernetes
url: https://github.com/kubernetes/kubernetes
language: Go
created_since: 80
updated_since: 0
contributor_count: 3712
org_count: 5
commit_frequency: 97.2
recent_releases_count: 79
updated_issues_count: 5094
closed_issues_count: 2868
comment_frequency: 5.6
dependents_count: 424194
criticality_score: 0.98594
```

```
criticality_score --repo github.com/kubernetes/kubernetes
name: kubernetes
url: https://github.com/kubernetes/kubernetes
language: Go
created_since: 80
updated_since: 0
contributor_count: 3712
org_count: 5
commit_frequency: 97.2
recent_releases_count: 79
updated_issues_count: 5094
closed_issues_count: 2868
comment_frequency: 5.6
dependents_count: 424194
criticality_score: 0.98594
```

Are you on latest 1.0.7 criticality_score ?

**Comment by Yikun** [comment]
@inferno-chromium Yes, it's latest 1.0.7, and get a wrong org_count = 10 [1] in k8s demo cmd, and then get the socore more than 1.

This problem will be fixed by #69. Thanks. And I will also close the alternate solution #70

[1] https://github.com/ossf/criticality_score/blob/main/criticality_score/run.py#L213
